### PR TITLE
promise chain dupe

### PR DESCRIPTION
**Pull request by LW001**

### CODE REVIEW

File: Commands/Commands/queue.js [hunk 1]
```diff
@@ -120,146 +120,142 @@ commands.dupe = {
...
-                          name: 'reverse',
-                          id: '322646981476614144'
-                        })
-                      }).catch(logger.raven)
-                    })
-                  }
+        uv.v1.loginAsOwner()
+        .then(c => c.get(`forums/${config.uservoice.forumId}/suggestions/${id}.json`)
+        .then((data) => c.get(`forums/${config.uservoice.forumId}/suggestions/${id2}.json`)
+        .then((data2) => msg.reply(`this will result in the following card.\n__Are you sure this is correct?__ (yes/no)`, false, {
```
Comment (ThaTiemsz) [inline Commands/Commands/queue.js:130]: These parentheses are reluctant

File: Commands/Commands/queue.js [hunk 2]
```diff
@@ -120,146 +120,142 @@ commands.dupe = {
...
+          url: data2.suggestion.url,
+          description: (data2.suggestion.text !== null) ? (data2.suggestion.text.length < 1900) ? data2.suggestion.text : '*Content too long*' : '*No content*',
+          fields: [{
+            name: 'Votes',
+            value: parseInt(data.suggestion.vote_count) + parseInt(data2.suggestion.vote_count)
+          }],
+          footer: {
+            text: (data2.suggestion.category !== null) ? data2.suggestion.category.name : 'No category'
+          }
+        }).then(() => wait(bot, msg)
```
Comment (ThaTiemsz) [inline Commands/Commands/queue.js:147]: Move this to a new line, don't forget the closing `)`

File: Commands/Commands/queue.js [hunk 3]
```diff
@@ -120,146 +120,142 @@ commands.dupe = {
...
-                cause: 'dupe_search_first',
-                message: (e.message !== undefined) ? e.message : JSON.stringify(e)
-              }, e)
-              msg.reply('an error occured, please try again later.').then(errmsg => {
-                setTimeout(() => bot.Messages.deleteMessages([msg, errmsg]), config.timeouts.errorMessageDelete)
-              })
-            }
-          })
+          }
+        })))).catch((e) => {
```
Comment (ThaTiemsz) [inline Commands/Commands/queue.js:230]: What in the world is this

File: Commands/Commands/queue.js [hunk 4]
```diff
@@ -120,146 +120,142 @@ commands.dupe = {
...
+            logger.log(bot, {
+              cause: 'dupe_search_first',
+              message: (e.message !== undefined) ? e.message : JSON.stringify(e)
+            }, e)
+            msg.reply('an error occured, please try again later.').then(errmsg => {
+              setTimeout(() => bot.Messages.deleteMessages([msg, errmsg]), config.timeouts.errorMessageDelete)
+            })
+          }
+        })
+        ).catch((e) => {
```
Comment (ThaTiemsz) [inline Commands/Commands/queue.js:245]: ­

**Review by ThaTiemsz — CHANGES_REQUESTED** [review]

**Review by nullpixel — CHANGES_REQUESTED** [review]
see tiemens review

**Comment by Arzte** [comment]
Closed in favor of ignoring fixing the current MB for doing the rewrite version and fixing up current MB.

**Pull request closed without merging**